### PR TITLE
feat: Change the header

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/common/header/Header.styles.ts [hunk 1]
```diff
@@ -4,39 +4,39 @@ import mixins from "@styles/mixins";
 import { Z_INDEX } from "@styles/zIndex";
 
 export const HeaderWrapper = styled.header`
-  position: fixed;
+  ${mixins.flexbox("column", "center", "center")};
   top: 0;
+  position: fixed;
   z-index: ${Z_INDEX.fixed};
+  max-width: 1920px;
```
Comment (jinoosss) [inline packages/web/src/components/common/header/Header.styles.ts:11]: If the browser size is large, this attribute will cause the header to be skewed to the left. 
Is there a reason for this attribute?

File: packages/web/src/constants/header.constant.ts [hunk 2]
```diff
@@ -16,3 +16,26 @@ export const HEADER_NAV = [
     path: "/dashboard",
   },
 ];
+
+export const SIDE_MENU_NAV = [
```
Comment (jinoosss) [inline packages/web/src/constants/header.constant.ts:20]: I think the Array type is not explicit.
For example, it is hard to predict that SIDE_MENU_NAV[0] is the value for 'Governance'.
How about declaring it as an object?

File: packages/web/src/components/common/search-menu-modal/SearchMenuModal.styles.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    width: 100%;
+    .recent-searches,
+    .popular-tokens {
+      ${mixins.flexbox("row", "center", "flex-start")};
+      padding: 16px 24px 8px 24px;
+      ${fonts.body12}
+      color: ${({ theme }) => theme.color.text04};
+    }
+  }
+  li {
```
Comment (jinoosss) [inline packages/web/src/components/common/search-menu-modal/SearchMenuModal.styles.ts:81]: Missing cursor attribute.

File: packages/web/src/components/common/search-menu-modal/SearchMenuModal.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <IconSearch className="search-icon" />
+          </SearchWrapper>
+        </SearchContainer>
+        <ModalContainer>
+          <ul>
+            <div className="recent-searches">Recent Searches</div>
+            {tokens
+              .filter(item => item.searchType === "recent")
+              .map((item, idx) => (
+                <li key={idx}>
```
Comment (jinoosss) [inline packages/web/src/components/common/search-menu-modal/SearchMenuModal.tsx:66]: The click event for the search item is missing.

**Review by jinoosss — COMMENTED** [review]
Nicely done.

I left a few comments.

File: packages/web/src/components/common/header/Header.styles.ts (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/common/header/Header.styles.ts:11]: It seems it was a mistake. Deleted. thank you !

File: packages/web/src/components/common/search-menu-modal/SearchMenuModal.styles.ts (hunk 3 above)
Comment (juntaepark) [reply packages/web/src/components/common/search-menu-modal/SearchMenuModal.styles.ts:81]: Added. thank you :joy:

File: packages/web/src/constants/header.constant.ts (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/constants/header.constant.ts:20]: good idea. I used object.

File: packages/web/src/components/common/search-menu-modal/SearchMenuModal.tsx (hunk 4 above)
Comment (juntaepark) [reply packages/web/src/components/common/search-menu-modal/SearchMenuModal.tsx:66]: Added!

**Review by jinoosss — APPROVED** [review]
LGTM !

**Pull request merged**